patches/components-permissions-android-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
-index 13ecd738384ecdc89a644b4abcaa96faed333336..9f4240d7ca46ec69c0ebe4a2f53c5e86f1026587 100644
+index 13ecd738384ecdc89a644b4abcaa96faed333336..199e974fc4a8555f51907413074e208a7be913e4 100644
 --- a/components/permissions/android/BUILD.gn
 +++ b/components/permissions/android/BUILD.gn
-@@ -115,6 +115,7 @@ android_library("java") {
-     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
-     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
+@@ -130,6 +130,7 @@ android_library("java") {
+     "//third_party/androidx:androidx_vectordrawable_vectordrawable_java",
+     "//ui/android:ui_java",
    ]
-+  sources += brave_components_permissions_java_sources
-   deps = [
-     ":core_java",
-     ":java_resources",
++  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
+   annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
```

Comment:
this should all be on one line and `"//base/android/jni_generator:jni_processor"` should be replaced by `brave_components_permissions_annotation_process_deps` or something along those lines

Reply:
the only one line that is added in the patch is `+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps`

Reply:
and even not added but increased from `+  sources += brave_components_permissions_java_sources` to `+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps`